### PR TITLE
#167190386 Setup base react app

### PR DESCRIPTION
**Pull request by chuxmykel**
#### What does this PR do?
- This PR sets up a react app from scratch using webpack.

#### Description of Task to be completed?
- setup react with webpack
- setup linting with eslint Airbnb style
- create an express server to serve the app
- create app.json file for review apps
- setup sass processing on the client

#### How should this be manually tested?
- pull the branch and run `npm run dev`
- Click on the links on the individual pages to test the routes

#### Any background context you want to provide?
- created components are just for test purposes and should be deleted in due time
- I would have added a screenshot but GitHub seems to be having some issues.

#### What are the relevant pivotal tracker stories?
- [#167190386](https://www.pivotaltracker.com/story/show/167190386)
- [#167190390](https://www.pivotaltracker.com/story/show/167190390)
- [#167190391](https://www.pivotaltracker.com/story/show/167190391)

#### Questions:
- Nil

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parserOptions": {
+    "sourceType": "module",
+    "allowImportExportEverywhere": true
+  },
+  "extends": "airbnb",
+  "globals": {
```
Comment (encodedBicoding) [inline .eslintrc.js:9]: Thank you very much Chuks for this great implementation, I have seen so much from this code that I do not know about previously. But I'm curious as to what this `globals` property does. can you fill me in?

**Review by Hadeneekeh — APPROVED** [review]
Great job @chuxmykel in such a short time.

LGTM!!!

**Review by encodedBicoding — APPROVED** [review]
LGTM

**Review by chiboycalix — APPROVED** [review]
This looks good. Great Job @chuxmykel

**Review by obayomi96 — APPROVED** [review]
Good job on this implementation. I like how you also Isolated the individual Webpack files which makes it easy to manage, we might not need to go into those files again. Great job on this task.

However, we might need to fix the Hound issues.

**Comment by chuxmykel** [comment]
I have specified es6 features in the eslint file now. I don't think we should have problems any more.

**Review by Lundii — APPROVED** [review]
LGTM

**Review by Hadeneekeh — APPROVED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (chuxmykel) [reply .eslintrc.js:9]: According to the [ESLint configuration page](https://eslint.org/docs/user-guide/configuring)
There are several pieces of information that can be configured:

Environments - which environments your script is designed to run in. Each environment brings with it a certain set of predefined global variables.
Globals - the additional global variables your script accesses during execution.
Rules - which rules are enabled and at what error level.
All of these options give you fine-grained control over how ESLint treats your code.

**Review by chiboycalix — APPROVED** [review]
LGTM

**Review by topseySuave — APPROVED** [review]
Nice setup, LGTM

**Pull request merged**